.github/CODEOWNERS

```diff
@@ -29,6 +29,9 @@ CMake/ @assignUser @majetideepak
 scripts/ @assignUser @majetideepak
 .github/ @assignUser @majetideepak
 
+# clear codeownership in these dirs
+velox/experimental/breeze
```

Comment:
Let's add @dreveman as the owner

Reply:
It only works if you have write access to the repo: 
![image](https://github.com/user-attachments/assets/6cca6c83-3e63-419a-9830-9b9d59306ad9)

Reply:
@pedroerp How do we grant write access?

Reply:
@Yuhta @assignUser added. Could you try again?

Reply:
I assume @dreveman needs to accept the invite as it still shows an error but should work once it shows the 'collaborator' tag

Reply:
I just accepted it

Reply:
:tada: 
![image](https://github.com/user-attachments/assets/13f62650-7587-4c14-b06d-53b33e0f005f)